src/Mod/TechDraw/App/Preferences.cpp

```diff
@@ -285,14 +285,14 @@ std::string Preferences::bitmapFill()
 
 double Preferences::GapISO()
 {
-    double factor = getPreferenceGroup("Dimensions")->GetFloat("GapISO", 8.0);
+    double factor = getPreferenceGroup("Dimensions")->GetFloat("GapISO", 0.0);
     return factor;
 }
 
 //! Returns the factor for calculating the ISO gap, not the actual distance.
 double Preferences::GapASME()
 {
-    double factor = getPreferenceGroup("Dimensions")->GetFloat("GapASME", 6.0);
+    double factor = getPreferenceGroup("Dimensions")->GetFloat("GapASME", 0.0);
```

Comment:
Wasn't it only ISO?

Reply:
This is the gap between the dimension points and the beginning of the extension line.  Both ISO and ASME allow a small gap in the interests of artistic freedom. ;)

The comment is wrong, though.  It belongs with the previous function.